controllers/suite_test.go

```diff
@@ -95,7 +98,7 @@ var _ = BeforeSuite(func() {
 	IT.Clientset, err = kubernetes.NewForConfig(IT.TestEnvironment.Config)
 	Expect(err).NotTo(HaveOccurred())
 
-	IT.TokenStorage, err = tokenstorage.New(IT.Client)
+	IT.VaultTestCluster, IT.TokenStorage = tokenstorage.CreateTestVaultTokenStorage(&testing.T{})
```

Comment:
Would `GinkgoT()` work here instead of `&testing.T{}`?

Reply:
omg, I was looking at this, thinking it is not possible because of this guy https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/testing/testing.go;l=766 only now I see that vault uses it's own interface created for this exact reason https://github.com/mitchellh/go-testing-interface/blob/master/testing.go#L18.

Thanks for making me looking at it again.

Reply:
fixed